<launch>
<arg name="dataFolder"/>
<param name="dataFolder" value="$(arg dataFolder)"/>


<node name="map_server" pkg="map_server" type="map_server"
args="$(arg dataFolder)FMap.yaml" output="screen"/>

<node pkg="tf" type="static_transform_publisher" name="tf_broadcaster" args="1 0 0 0 0 0 1 world map 100" />

<node pkg="data_processing" type="PickleBroadcastNode.py" name="PickleBroadcastNode" output="screen"/>

  
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find data_processing)/rviz/gtviz.rviz"/>
</launch>
